assets/section-footer.css

```diff
@@ -466,11 +467,40 @@ noscript .localization-selector.link {
   max-width: 20rem;
 }
 
+.copyright__content {
+  font-size: 1rem;
+}
+
 .copyright__content a {
   color: currentColor;
   text-decoration: none;
 }
 
+.policies {
+  display: inline;
+}
+
+.policies li {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.policies li::before {
+  content: '\00B7';
```

Comment:
Ive seen this being used in some other places in the theme like blog. I wonder if we can re-use the class? And just update the spacing?
 https://screenshot.click/08-42-9mpqo-wa4u8.png

Reply:
That's a different symbol though.

We're using a [`Middle dot` character](https://www.compart.com/en/unicode/U+00B7) while the blog is using a [`Bullet`](https://www.compart.com/en/unicode/U+2022) character. I'm not sure if they're meant to be using the same, though. For policy, we definitely want to keep it as `\00B7` because of it's sizing.